tests/functional_test_base.go

```diff
@@ -199,7 +200,7 @@ func ApplyTestClusterParams(options []Option) TestClusterParams {
 // If the Logger is not set, this method creates a new log.TestLogger which logs to stdout and stderr.
 func (s *FunctionalTestBase) setupLogger() {
 	if s.Logger == nil {
-		s.Logger = log.NewNoopLogger()
+		s.Logger = log.NewTestLogger()
```

Comment:
I remember we started to use `NoopLogger` some time ago. I actually find convenient to use `TestLogger` while debugging and always make similar change. I am all for it, to go back to `TestLogger` again.

Reply:
Yeah, I'm not sure why/when that happened.. I think tests should always log. We can have env vars or whatever to control test logging (I think we have some already)